genesis/utils/usd.py

```diff
@@ -0,0 +1,312 @@
+                                metallic_image = mu.PIL_to_array(Image.open(metallic_texture_path))
+                                metallic_texture = mu.create_texture(metallic_image, None, "linear")
+                                gs.logger.info(f"Loading Metallic texture: {metallic_texture_path}")
+
+                        # update surface texture
+                        mesh_surface.update_texture(
+                            color_texture=color_texture,
+                            normal_texture=normal_texture_path,
+                            opacity_texture=opacity_texture,
+                            roughness_texture=roughness_texture,
+                            metallic_texture=metallic_texture,
+                        )
+
+                    else:  # load from .mdl description
+                        source = shader.GetImplementationSource()
+                        # handle the mdl shader source from asset
+                        if source == UsdShade.Tokens.sourceAsset:
+                            mdl_asset_attr = shader.GetPrim().GetAttribute("info:mdl:sourceAsset")
+                            mdl_sub_identifier_attr = shader.GetPrim().GetAttribute(
+                                "info:mdl:sourceAsset:subIdentifier"
+                            )
+
+                            if mdl_asset_attr and mdl_sub_identifier_attr:
+                                mdl_asset_path = mdl_asset_attr.Get().resolvedPath  # absolute path
+                                mdl_material_name = mdl_sub_identifier_attr.Get()
+                                diffuse_texture_path, normal_texture_path, diffuse_tint, orm_texture_path = (
+                                    extract_texture_paths_from_mdl(mdl_asset_path)
+                                )
+
+                                if diffuse_tint:
+                                    diffuse_tint = np.array(list(diffuse_tint) + [1.0], dtype=float)  # RGB + A
+                                else:
+                                    diffuse_tint = np.ones(4, dtype=float)
+
+                                color_texture = None
+                                normal_texture = None
+                                opacity_texture = None
+                                roughness_texture = None
+                                metallic_texture = None
+
+                                if diffuse_texture_path:
+                                    diffuse_texture_path = os.path.normpath(
+                                        os.path.join(os.path.dirname(mdl_asset_path), diffuse_texture_path)
+                                    )
+                                    diffuse_image = mu.PIL_to_array(Image.open(diffuse_texture_path))
+                                    color_texture = mu.create_texture(diffuse_image, diffuse_tint, "srgb")
+                                    gs.logger.info(f"Get Diffuse texture: {diffuse_texture_path}")
```

Comment:
All these trace should be `gs.logger.debug` not `gs.logger.info`, so that they are hidden by default. Let's make sure that critical info stands out of the noise.